lib/tsconfig/TsConfigSyntax.c

```diff
@@ -8,8 +8,8 @@
 
 #define FLEX_SCANNER
 #define YY_FLEX_MAJOR_VERSION 2
-#define YY_FLEX_MINOR_VERSION 6
-#define YY_FLEX_SUBMINOR_VERSION 0
+#define YY_FLEX_MINOR_VERSION 5
+#define YY_FLEX_SUBMINOR_VERSION 37
```

Comment:
i wondered WTH that's checked in for.   In normal practice we don't check in *.o or Makefile.

Accidentally in there.